streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamKTableLeftJoinTest.java

```diff
@@ -199,4 +207,65 @@ public void shouldClearTableEntryOnNullValueUpdates() {
                 new KeyValueTimestamp<>(3, "XX3+Y3", 3));
     }
 
+    @Test
+    public void shouldNotDropLeftNullKey() {
```

Comment:
Adding two missing test cases -- does not expose any bug -- for stream-table-join `mappedkey == key` so not dropping on stream-null-key did work as expected